contrib/jinzhu/gorm/gorm_test.go

```diff
@@ -281,3 +281,26 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestContext(t *testing.T) {
+	sqltrace.Register("postgres", &pq.Driver{})
+	db, err := Open("postgres", "postgres://postgres:postgres@127.0.0.1:5432/postgres?sslmode=disable")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	t.Run("with", func(t *testing.T) {
+		// should not use basic type string as key in context.WithValue (golint)
```

Comment:
to fix `contrib/jinzhu/gorm/gorm_test.go:294:14:warning: should not use basic type string as key in context.WithValue (golint)`

let me know if you'd prefer just adding a `// nolint` to line 296 and pass a string key

Reply:
No this is fine. What you did is perfect. I think it’s also fine to remove this comment. Once you do that we’re good to merge.